core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -196,6 +196,12 @@ angular.module('oppia').controller(
 
         [$scope.activeSuggestionId, $scope.activeContribution] = (
           remainingContributions.pop());
+
+        if (!$scope.activeContribution.details) {
```

Comment:
Can we add a comment mentioning the corresponding issue number explaining why we're doing this?

I think ideally we'd probably want to advance to the next valid suggestion, but because this issue should almost never occur in prod, I think this is fine.

Reply:
done